ingestion/src/metadata/utils/dbt_config.py

```diff
@@ -182,18 +182,30 @@ def _(config: DbtCloudConfig):  # pylint: disable=too-many-locals
         runs_data = response.get("data")
         if runs_data:
             run_id = runs_data[0]["id"]
-            logger.debug("Requesting [dbt_catalog]")
-            dbt_catalog = client.get(
-                f"/accounts/{account_id}/runs/{run_id}/artifacts/{DBT_CATALOG_FILE_NAME}"
-            )
+            try:
+                logger.debug("Requesting [dbt_catalog]")
+                dbt_catalog = client.get(
+                    f"/accounts/{account_id}/runs/{run_id}/artifacts/{DBT_CATALOG_FILE_NAME}"
+                )
+            except Exception as exc:
+                logger.info(
```

Comment:
I think this should be `warning` instead of `info`

Reply:
@NiharDoshi99 it should be info, since it is a optional thing we only want to inform users that catalog file is not passed

Reply:
Oh Ok 👍